build/Rulesets/NonShippingProject_BuildRules.ruleset

```diff
@@ -33,4 +33,17 @@
     <!-- Mark constant field as 'const' instead of static readonly - not useful for tests -->
     <Rule Id="CA1802" Action="None" />
   </Rules>  
+
+  <Rules AnalyzerId="Xunit.Analyzers" RuleNamespace="Xunit.Analyzers">
+    <!-- Do not use Enumerable.Any() to check if a value exists in a collection -->
+    <Rule Id="xUnit2012" Action="None" />
```

Comment:
Can we file bugs for all of these cases?  Seems like these are hiding real issues

Reply:
Sure.